ChangeLog-split.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+
+# https://stackoverflow.com/questions/59895/how-to-get-the-source-directory-of-a-bash-script-from-within-the-script-itself
+SOURCE="${BASH_SOURCE[0]}"
+DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
```

Comment:
Why not just use?
```
dirname "$0"
```

Reply:
Please see link in comment just above - https://stackoverflow.com/questions/59895/how-can-i-get-the-source-directory-of-a-bash-script-from-within-the-script-itsel
In short: it resolves symlinks and gives real path. I am not sure if that matters for case of this script, just use this solution everywhere to be safe. ^__^